### DISCUSSION TITLE
Just got amd64-cuda-viseron:modularize working on Unraid

### DISCUSSION THREAD

**Discussion by JeffOnGithub** [roflcoopter/viseron#363] [General]
Needed some modification to the stock version : https://forums.unraid.net/topic/96774-support-viseron-a-self-hosted-nvr-with-object-detection/

I currently have 2 RTSP cameras, stream decoding and AI working on the Nvidia GPU with almost no cpu load, pushing events to Mosquitto, recording on detection, nice! 

I've been really disappointed by Frigate's performance on CPU (and no GPU support) which brought me here, great work!

If this can help someone, here is the configuration in Unraid Docker i used

![image](https://user-images.githubusercontent.com/8843414/188025309-c16a7ea1-a3b1-4208-aa6e-edae176fb393.png)

For the config.yml file it's almost the example file with the following addition to get the decoding on the GPU 
      width: 2560
      height: 1440
      fps: 5
      codec: h264_cuvid

**Comment by roflcoopter** [comment]
Interesting, so you're saying that it did not automatically detect your GPU if you didnt specify `codec` in the config?
